lib/dry/inflector.rb

```diff
@@ -273,7 +273,7 @@ def uncountable?(input)
 
     # @since 0.1.0
     # @api private
-    ORDINALIZE_TH = (4..16).to_set.freeze
+    ORDINALIZE_TH = (4..16).each_with_object({}) { |n, ret| ret[n] = true }.freeze
```

Comment:
Basically `Set` is the same as having key: true hash. And we are using it in the `Inflections` for performance reasons. So, why removing it?

Reply:
Agree @Ptico.
The only fix needed is the
```ruby
else "#{number}th"
```

By the way... this bug affects Inflecto/Flexus!!! 😱